airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -522,6 +527,30 @@ class MetafieldProductImage(Metafield):
     }
     """
 
+    @property
+    def query_nodes(self) -> List[Field]:
+        """
+        This is the overide for the default `query_nodes` method,
+        because the usual way of retrieving the metafields for product images` was suddently deprecated,
+        for `2024-10`, but the changes are reflected in the `2024-04` as well, starting: `2024-08-01T00:06:44`
+
+        CAT Test (FAILED): https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/nightly_builds/master/1722470804/269f22e245fc60c872ec96c6eba20cb7bf72b4d4/source-shopify/2.4.16/output.html
```

Comment:
How long do we keep those reports? If they are not accessible after some time, is there some information we should move in this comment?

Reply:
No idea, how long this is storred. Removed.